Laboratory/Tests/L4-Standard-Library/ElementTimeSpan.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Laboratory.Tests.L4.StandardLibrary
+{
+    internal class ElementTimeSpan : StandardLibraryFixture
```

Comment:
New tests should be written using ExpressionEvaluation or FunctionEvaluation to support both compile-time and runtime cases. See Numbers and Boolean tests in L3-Prelude for an example.

Reply:
Done :+1: